frontend/model/state.js

```diff
@@ -477,8 +483,8 @@ const getters = {
   },
   groupMembersSorted (state, getters) {
     const profiles = getters.currentGroupState.profiles
-    if (!profiles) return []
-    const weJoinedMs = new Date(profiles[getters.ourUsername].joinedDate).getTime()
+    if (!profiles || !profiles[getters.ourUsername]) return []
+    const weJoinedMs = new Date(profiles[getters.ourUsername]?.joinedDate).getTime()
```

Comment:
Since line 486 verifies that `profiles[getters.ourUsername]` exists, I don't think we need the `?` after `profiles[getters.ourUsername]` on line 487, right?

Reply:
Correct. Nice catch.